### PR TITLE
Transfer documentation of tracking states into repo

### PR DESCRIPTION
**Pull request by nerdstrike**
I have lifted the content from your Google doc and editorialised it slightly.

### CODE REVIEW

File: docs/pipeline_states.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+| analysis pending | Run is ready to be picked up by the analysis daemon |
+| analysis in progress | Run has been automatically submitted for analysis. Stage 1 analysis is in progress |
+| secondary analysis in progress | All data are now in stage 2 of the analysis - secondary pipelines such as for Heron, "robo-QC" |
+| analysis on hold | Something went wrong either during analysis or in Manual QC. Investigating, probably re-analysing |
+| analysis complete | Analysis has finished |
+| qc review pending | Run is ready for manual QC |
+| qc in progress | Manual QC in progress (Data team in Ops) |
+| qc on hold | Problem detected, investigation in progress by the QC team. They may involve NPG or PSD to learn more. Change to LIMS data in Sequencescape and ml warehouse and re-analysis might be required |
+| archival pending | Run is ready to be archived, waiting to be picked up by the archival daemon. Normally run is automatically transferred to this status once manual QC of all lanes of the run is complete. However, UKBMP runs are transferred to this status straight after ‘analysis complete’, bypassing the manual QC step. |
```
Comment (mgcam) [inline docs/pipeline_states.md:24]: 'UKBMP runs' -> 'Heron runs'

File: docs/pipeline_states.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+| archival in progress | Run has been submitted for archival. Data in our QC database and ml-warehouse will be uploaded/updated, data are uploaded to/updated in iRODS, or to the appropriate offsite location, e.g. AWS / Seven Bridges |
```
Comment (mgcam) [inline docs/pipeline_states.md:25]: Remove ', e.g. AWS / Seven Bridges'? This was UKB specific, is not done any longer

File: docs/pipeline_states.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+| run archived | Archival to iRODS is finished |
+| qc complete | NPG hands the data off. End users can safely access it from iRODS/warehouse etc. Once this status is reached, staging data becomes eligible for deletion after a grace period. Faculty data is allowed to remain for two weeks after completion |
+
+## See also
+
+[ML Warehouse guide](https://docs.google.com/document/d/1CWJBvVEiHSnd1CnmjvCDvQxv17ITVxd_Bsxq77cTDrs/edit#)
```
Comment (mgcam) [inline docs/pipeline_states.md:31]: Not accessible publicly. Not really important for tracking. I'd remove

File: docs/pipeline_states.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+[Analysis pipeline flow](https://github.com/wtsi-npg/npg_seq_pipeline/blob/devel/data/config_files/function_list_central.json.png)
```
Comment (mgcam) [inline docs/pipeline_states.md:32]: Shall we add archival pipeline flow as well?

File: docs/pipeline_states.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
+# Run States
+
+The Tracking interface shows a sequence of states for Illumina runs. They have the following meanings in approximate order of execution:
+
+| Pipeline state | Meaning |
```
Comment (mgcam) [inline docs/pipeline_states.md:5]: Pipeline state -> Run state (status) ?

**Review by mgcam — CHANGES_REQUESTED** [review]
The text needs a few changes, some parts of the text are out of date out of date in the original document. Should this file be called run_states.md? Would be good to add a link to this file to the main README.md file

File: docs/pipeline_states.md (hunk 2 above)
Comment (nerdstrike) [reply docs/pipeline_states.md:25]: If I remove it, the feature will surely rise again! Removed

**Review by mgcam — APPROVED** [review]

**Pull request merged**